src/app/components/Form/AmountInput/index.tsx

```diff
@@ -10,7 +10,7 @@ import { stringToFixedPrecision } from 'utils/display-text/format';
 
 interface Props {
   value: string;
-  onChange: (value: string) => void;
+  onChange: (value: string, isTotal?: boolean | undefined) => void;
```

Comment:
You don't need to specify that `isTotal` can be either `boolean` or `undefined` since `?` already does that. So just remove `| undefined`.